src/data/akData.ts

```diff
@@ -0,0 +1,17 @@
+export const AK_DATA = {
+  phone: "+48 505 248 666",
+  email: "ak.debica@gmail.com",
+  address: "39-200 Dębica, ul. Rzeszowska 15",
+  socialMedia: {
+    facebook: "https://www.facebook.com/Szlak.Partyzancki",
+    instagram: "https://www.instagram.com", // Todo: Update link https://app.clickup.com/t/8697u6n5a
+    youtube: "https://www.youtube.com/@armiakrajowadebica3809",
+    x: "https://x.com/szlak",
+  },
+  bankName: "Santander Bank Polska",
+  bankAccountNumber: "41 1090 2590 0000 0001 5299 0666",
```

Comment:
Could be grouped under `bank` or something with `bankName` and `bankAccountNumber`